README.md

```diff
@@ -31,9 +31,9 @@ Detailed version can be seen on [codecov.io][product-cov-link].
 [explorer-nodewatch-cov-link]: https://codecov.io/gh/symbol/product/tree/dev/explorer/nodewatch
 
 [faucet-authenticator-job]: https://jenkins.symboldev.com/blue/organizations/jenkins/Symbol%2Fgenerated%2Fproduct%2Ffaucet-authenticator/activity?branch=dev
-[faucet-authenticator-lint]: https://jenkins.symboldev.com/buildStatus/icon?job=Symbol%2Fgenerated%2Fproduct%2Ffaucet-authenticator%2Fdev%2F&config=faucet-authenticator-lint
-[faucet-authenticator-test]: https://jenkins.symboldev.com/buildStatus/icon?job=Symbol%2Fgenerated%2Fproduct%2Ffaucet-authenticator%2Fdev%2F&config=faucet-authenticator-test
-[faucet-authenticator-cov]: https://codecov.io/gh/symbol/product/branch/dev/graph/badge.svg?token=SSYYBMK0M7&flag=faucet-authenticator
```

Comment:
I think the issue is that since you are using ``authenticator`` as the packageId then codecov flag needs to match the packageId.

https://github.com/symbol/product/blob/d2dd5393af84194f6922c923ba27cb7b16023e3a/.github/codecov.yaml#L75

Or the easier fix is to update the packageId in the Jenkinsfile from `` authenticator`` to `` faucet-authenticator`` since you are using `` faucet-authenticator`` everywhere else.

Reply:
I did change that before, but the icon is showing unknowns as well.

Reply:
you need to check the status on the current branch ``faucet/fix-package-id``
https://codecov.io/gh/symbol/product/branch/faucet/fix-package-id/graph/badge.svg?token=SSYYBMK0M7&flag=faucet-authenticator